samples/dymaptic.GeoBlazor.Core.Sample.Server/Pages/_Layout.cshtml

```diff
@@ -16,7 +16,6 @@
         <link href="_content/dymaptic.GeoBlazor.Core.Sample.Shared" />
         <script src="_content/dymaptic.GeoBlazor.Core.Sample.Shared/functions.js"></script>
         <component type="typeof(HeadOutlet)" render-mode="ServerPrerendered" />
-        <link rel="stylesheet" href="https://js.arcgis.com/4.24/esri/themes/light/main.css">
```

Comment:
Don't we still need to load this CSS file? How else would the person set the theme?

Reply:
I see, later we are defaulting to the `dark` theme in `core.css`. I think we need a way for the user to control which CSS theme is loaded.

Reply:
I put these back into link tags, after trying several other approaches, but with a local path to the bundled css